Classes/Cache/ExposedValueObjectCache.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace Netlogix\JsonApiOrg\AnnotationGenerics\Cache;
+
+use Neos\Flow\Annotations as Flow;
+use Netlogix\JsonApiOrg\AnnotationGenerics\Domain\Model\ExposedValueObjectInterface;
+
+/**
+ * @Flow\Proxy(false)
+ */
+class ExposedValueObjectCache
```

Comment:
This class could be a `@Flow\Scope("singleton")` without static methods and properties